### DISCUSSION TITLE
Deprecated libxml function as of PHP 8

### DISCUSSION THREAD

**Discussion by anoymouserver** [alexdebril/feed-io#327] [General]
I just randomly noticed that [`libxml_disable_entity_loader`](https://www.php.net/manual/en/function.libxml-disable-entity-loader.php) was marked as deprecated with PHP 8 and it is used here
https://github.com/alexdebril/feed-io/blob/96ca6d03eaf5e638273248c07a348ed76148bc25/src/FeedIo/Explorer.php#L55-L64

Possible solution: https://www.php.net/manual/en/function.libxml-disable-entity-loader.php#125661
(Just as information, in case you didn't already know 🙂)

**Comment by alexdebril** [comment]
Thanks, although it's the goal of the libxml's version test 🙂

I noticed that a few months back when I prepared feed-io's compatibility with PHP 8. If you look at the code history you'll see the commit with the successful build on Travis.
